AdyenCard/Form/FormCardSecurityCodeItem.swift

```diff
@@ -69,6 +74,20 @@ extension Array where Element == CardBrand {
         }
     }
     
+    /// At the moment, due to UI concerns, we won't hide cvc/exp date fields even when response is hidden
```

Comment:
maybe this `extension Array where Element == CardBrand` can be in a separate file, since now it contains things not related to `FormCardSecurityCodeItem`